### PR TITLE
Use #map instead of #pluck

### PR DESCRIPTION
**Pull request by baphled**

### CODE REVIEW

File: lib/tasks/generate_popular_articles.rake [hunk 1]
```diff
@@ -3,5 +3,5 @@ task :generate_popular_articles => :environment do
   require File.join(Rails.root, 'lib', 'popular_article')
 
   most_popular_articles = MostPopularArticle.last || MostPopularArticle.new
-  most_popular_articles.update_attribute(article_ids: PopularArticle.find.pluck(:id))
+  most_popular_articles.update_attribute(article_ids: PopularArticle.find.map(&:id))
```
Comment (benlovell) [inline lib/tasks/generate_popular_articles.rake]: Que?

**Comment by esbel** [comment]
:+1:

File: lib/tasks/generate_popular_articles.rake (hunk 1 above)
Comment (baphled) [inline lib/tasks/generate_popular_articles.rake]: It's no an AR model. Though, I guess I should use a better word than find

Comment (benlovell) [inline lib/tasks/generate_popular_articles.rake]: Ah :eyes:

**Comment by benlovell** [comment]
:ship: that :hankey:

**Pull request merged**